Porting/bisect-runner.pl

```diff
@@ -958,6 +958,45 @@ =head2 When did perl start failing to build on a certain platform using C<g++> a
 
 =back
 
+=head2 When did a test file start to emit warnings?
+
+=over 4
+
+=item * Problem
+
+While F<make test> was running, warnings not previous seen were being emitted
+when F<dist/Tie-File/t/43_synopsis.t> was being run.  At what commit were
+these warnings first emitted?
+
+=item * Solution
+
+We know that when this test file was first committed to blead, no warnings
+were observed and there was no output to C<STDERR>.  So that commit becomes
+the value for C<--start>.
+
+Since the test file in question is for a CPAN distribution maintained by core,
+we must prepare to run that test by including C<--target=test_prep> in the
+bisection invocation.  We then run the test file in a way that captures
+C<STDERR> in a file.  If that file has non-zero size, then we have presumably
+captured the newly seen warnings.
+
+    export ERR="/tmp/err"
+
+    .../perl Porting/bisect.pl \
+        --start=507614678018ae1abd55a22e9941778c65741ba3 \
+        --end=d34b46d077dcfc479c36f65b196086abd7941c76 \
+        --target=test_prep \
+        -e 'chdir("t"); system("./perl harness ../dist/Tie-File/t/43_synopsis.t 2>$ENV{ERR}"); -s $ENV{ERR} and die q<$ENV{ERR} has non-zero size, suggesting warnings thrown>;'
```

Comment:
You can break up this long line. Since it's all a single-quoted string, you won't need backslashes:
```
        -e '
            chdir("t");
            system("./perl harness ../dist/Tie-File/t/43_synopsis.t 2>$ENV{ERR}");
            -s $ENV{ERR} and die q<$ENV{ERR} has non-zero size, suggesting warnings thrown>
        '
```
A smaller indent and a briefer die message should then be enough to avoid the long-line exception.